docs/testing/coverage.md

```
@@ -163,17 +167,17 @@ reportgenerator -reports:"coverage/**/*.opencover.xml" -targetdir:"coveragerepor
 
 ## 📊 Exemplos de Relatórios
 
-### **Relatório de Sucesso (≥85%)**
+### **Relatório de Sucesso (≥90%)**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)